.github/workflows/hierarchy.yml

```diff
@@ -156,3 +114,10 @@ jobs:
       - license-check
       - typos
     uses: ./.github/workflows/_move_ide.yml
+
+  split-cluster:
```

Comment:
This is missing from the Confluence page

Reply:
No it isn't 😆

Reply:
It's in nightly, not the hierarchy

Reply:
It's in both!

Reply:
Ahhhh, my search didn't find it because it's `split_cluster` and not `split-cluster`!

Reply:
fixed it